src/ui/public/courier/data_source/build_query/from_lucene.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash';
+
+export function buildQueryFromLucene(queries, decorateQuery) {
```

Comment:
Fo sho. Since I'll be combining the work here with my next PR before merging anything, I may add these in a big test writing frenzy at the end of the week.